client/src/overview/subComponents/StyleSelector.jsx

```diff
@@ -27,8 +27,8 @@ function StyleSelector({ stylePics, currentProductStyle }) {
         {`Style > ${currentStyleName}`}
       </div>
       <div className='styleThumbnail'>
-        {stylePics
-          ? stylePics.map((stylePic, index) => {
+        {thumbnailPics
+          ? thumbnailPics.map((stylePic, index) => {
             return (
             <span key={index} className={`thumbnail tn${index}`} onClick={ () => thumbnailClicked(event, index) }>
```

Comment:
Another instance of index on the key, this is me just being nit-picky. It probably won't effect the functionality of the app but may not be a bad idea to implement best practices.